src/codegen/enums.rs

```diff
@@ -57,6 +57,10 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     }
     imports.add("glib::translate::*", None);
```

Comment:
Should probably also not any imports when not generating this file anyway

Reply:
You right, `if !has_any {return}` need be sooner